script/bootstrap

```diff
@@ -1,16 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
 # script/bootstrap: Resolve all dependencies that the application requires to
 #                   run.
 
 set -e
 
-cd "$(dirname "$0")/.."
+cd "${0%/*}/.."
```

Comment:
I'm not sure, IMHO, parameter expansion is the right and idiom way to do this task.

See also [my answer](http://unix.stackexchange.com/a/254910/38906) and the link I gave there for more details.